src/network-config.js

```diff
@@ -0,0 +1,42 @@
+const getNetworkConfig = function getNetworkConfig(networkId) {
+    switch (networkId) {
+        case 'mainnet':
+            return {
+                nodeUrl: 'https://rpc.mainnet.near.org',
+                networkId: 'mainnet',
+                evmAccountId: 'evm',
+                walletUrl: 'https://wallet.near.org',
+                explorerUrl: 'https://explorer.near.org',
+            };
+        case 'testnet':
+            return {
+                nodeUrl: 'https://rpc.testnet.near.org',
+                networkId: 'default',
+                evmAccountId: 'evm',
+                walletUrl: 'https://wallet.testnet.near.org',
+                explorerUrl: 'https://explorer.testnet.near.org',
+            };
+        case 'betanet':
+            return {
+                nodeUrl: 'https://rpc.betanet.near.org',
+                networkId: 'betanet',
+                evmAccountId: 'evm',
+                walletUrl: 'https://wallet.testnet.near.org',
+                explorerUrl: 'https://explorer.testnet.near.org',
```

Comment:
Looks like `walletUrl` and `explorerUrl` are incorrect, here, for betanet.

Reply:
Good eye! pushed the fix. thank you